src/containers/FooterContainer/footerasset.json

```diff
@@ -0,0 +1,122 @@
+{
```

Comment:
If I understood correctly, this is currently an unused file?

Reply:
Yep, removed ✅